embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/session/SessionHandler.kt

```diff
@@ -207,6 +207,7 @@
 
     private fun stopPeriodicSessionCaching() {
         logger.logDebug("Stopping session caching.")
+        sessionPeriodicCacheExecutorService.shutdown()
```

Comment:
Do we ever start the caching again once we stop? Cancelling the future should be enough if we ever want to start this process again.